go.mod

```diff
@@ -80,7 +80,7 @@ require (
 	github.com/google/go-containerregistry v0.20.2
 	github.com/iamacarpet/go-win64api v0.0.0-20210311141720-fe38760bed28
 	github.com/k3s-io/helm-controller v0.15.15
-	github.com/k3s-io/k3s v1.29.13-0.20250113231425-cba30eb6b361 // release-1.29
+	github.com/k3s-io/k3s v1.29.13-rc1.0.20250116004245-c577569268df // release-1.29
```

Comment:
just checking, do we want the -rc version here?

Reply:
that's just the pseudo-version that go has assigned to the latest commit on that branch. Since we no longer wait to tag K3s releases prior to each rke2 release we wont necessarily see a full release here.